CHANGES.rst

```diff
@@ -370,6 +370,9 @@ astropy.uncertainty
 astropy.units
 ^^^^^^^^^^^^^
 
+- Use proper precision when printing a ``Quantity`` in case precision format not
```

Comment:
Maybe "when creating a string from a ``Quantity``"? It affects `to_string()` and `__str__`.

Reply:
Should be fixed (and migrated to milestone 4.0.5) in https://github.com/astropy/astropy/pull/11145/commits/acb7cc342df621b71899ff96c02843f6c31ff4bd